### DISCUSSION TITLE
Local picture display problem

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#888] [Questions and Help] [answered]
Hello, because some pictures don't want to be placed on the server, can lowdefy's picture block display local pictures?

**Comment by Gervwyk** [comment, marked as answer]
Hi @xiaoxiji1234 

Yes, if you place the picture in the `public` folder, you should be able to display it. Remember this will host the image files as public files, accessible to anyone with access to the server network.

Am I understanding your question correctly? What is your lowdefy config like for the picture?

**Reply by xiaoxiji1234** [reply]
Hi, The picture configuration is like this
`
- id: img
   type: Img
   properties:
     src: ./1.webp
`
I tried to put the picture into the public folder, but it still didn't show the picture. Was the SRC path written wrong?

**Reply by xiaoxiji1234** [reply]
Sorry, I've solved it. It's the problem of path. Thank you
